docs/content/developers/index.md

```diff
@@ -0,0 +1,56 @@
+# Technische documentatie voor developers
+
+De standaard wordt opgebouwd op basis van drie componenten, elk met hun eigen
+OAS 3.0 definities. Het gaat dan om de zaakregistratiecomponent (ZRC),
+documentregistratiecomponent (DRC) en de zaaktypecatalogus (ZTC).
+
+De ontwikkeling van iedere component vindt plaats in een aparte git repository:
+
+* [ZRC](https://github.com/vng-Realisatie/gemma-zaakregistratiecomponent)
+* [DRC](https://github.com/VNG-Realisatie/gemma-documentregistratiecomponent)
+* [ZTC](https://github.com/VNG-Realisatie/gemma-zaaktypecatalogus)
+
+## Ontwikkelen van een applicatie die van deze componenten gebruik maakt
+
+Bij het ontwikkelen van vakapplicaties (ook wel afhandelapplicaties genoemd),
+kunnen deze referentie-implementaties ingezet worden als backend services.
+
+### Services starten
+
+Alle componenten zijn als Docker containers beschikbaar.
+
+In `infra` staat een bruikbare `docker-compose.yml` docker-compose file.
+Hiervoor heb je nodig:
+
+* Docker
+* Docker-compose
+
+Voorbeeld gebruik:
+
+```bash
+cd infra
+docker-compose up -d
+```
+
+Je kan nu de componenten bereiken:
```

Comment:
Wennen hoor dat Nederlands...

Zie voor de uitleg Docker van het ZTC: https://github.com/VNG-Realisatie/gemma-zaaktypecatalogus/blob/master/INSTALL.rst
Denk dat dit een goede opzet is. Vooral puntje 4 is denk ik handig, voor als je docker-machine gebruikt.

Reply:
Ik zal even spieken wat ik nog kan overnemen :+1: